clustermesh-apiserver/main.go

```diff
@@ -554,8 +554,8 @@ func synchronizeCiliumEndpoints(clientset k8sClient.Clientset) {
 
 func startServer(startCtx hive.HookContext, clientset k8sClient.Clientset, services resource.Resource[*slim_corev1.Service]) {
 	log.WithFields(logrus.Fields{
-		"cluster-name": cfg.clusterName,
```

Comment:
This is not a 1:1 change, as `logfields.ClusterName` is `clusterName`, not `cluster-name`. This can break scripts which parse output

Reply:
```
	cilium/pkg/option/config.go
// ClusterName is the name of the ClusterName option
	ClusterName = "cluster-name"
```

actually logfields.ClusterName ==  "cluster-name"